hyperdome/client/scripts/start_client.py

```diff
@@ -19,19 +19,45 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-import sys
+import logging
 
+from autologging import install_traced_noop
 import click
 
-from ...common.common import version
+logging.addLevelName(1000, "OFF")
 
 
 @click.command()
-@click.option("--debug", "-d", is_flag=True)
-@click.version_option(version, prog_name="Hyperdome Server")
-def start(debug):
-    if debug:
-        setattr(sys, "hyperdome_dev_mode", True)
+@click.option(
+    "--log-level",
+    "-l",
+    "log_level",
+    type=click.Choice(
+        ["TRACE", "DEBUG", "INFO", "WARNING", "ERROR", "OFF"], case_sensitive=False
+    ),
+    default="ERROR",
+    help="override logging level for this run",
```

Comment:
Capitalization doesn't really matter for logging, but it should probably be better for help messages

Reply:
#89